content/en/agent/amazon_ecs/apm.md

```diff
@@ -96,14 +96,14 @@ For more examples of setting the Agent hostname in other languages, refer to the
 
 ```javascript
 const tracer = require('dd-trace');
+tracer.init();
```

Comment:
The init can also be on the same line as the require, like:

`const tracer = require('dd-trace').init();`

Reply:
done!